### PR TITLE
Use log functions of core framework on test/e2e/apimachinery

### PR DESCRIPTION
**Pull request by clarklee92**
Signed-off-by: clarklee92 <clarklee1992@hotmail.com>

**What type of PR is this?**
> /kind cleanup

**What this PR does / why we need it**:
This makes sub packages of e2e test apimachinery to use log functions
of core framework instead for avoiding circular dependencies.

**Which issue(s) this PR fixes**:

Ref: #81427

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by alejandrox1 — COMMENTED** [review]
@clarklee92 thank you for working on this!

/lgtm
/ok-to-test
/priority important-soon
/area e2e-test-framework

/assign @oomichi @timothysc @andrewsykim
for final approval

File: test/e2e/apimachinery/chunking.go [hunk 1]
```diff
@@ -69,9 +68,9 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 				if err == nil {
 					return
 				}
-				e2elog.Logf("Got an error creating template %d: %v", i, err)
+				framework.Logf("Got an error creating template %d: %v", i, err)
 			}
-			e2elog.Fail("Unable to create template %d, exiting", i)
+			framework.Fail("Unable to create template %d, exiting", i)
```
Comment (alejandrox1) [inline test/e2e/apimachinery/chunking.go]: /lgtm cancel
So it seems like there is no `framework.Fail()` method.
Could you please add this method to https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/log.go

Here is the defintion of `e2elog.Fail()`:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/log/logger.go#L57 to

**Comment by alejandrox1** [comment]
@clarklee92 could you please remove the `Fixes #81427` from your PR message ?
/hold

**Comment by oomichi** [comment]
Nice point @alejandrox1 
It is nice to use "Ref: #xxx" instead of "Fixes: #xxx" in this case.

File: test/e2e/apimachinery/chunking.go (hunk 1 above)
Comment (clarklee92) [reply test/e2e/apimachinery/chunking.go]: Thanks for pointing out, already updated.

**Comment by clarklee92** [comment]
> @clarklee92 could you please remove the `Fixes #81427` from your PR message ?
> /hold

Thanks, updated.

**Comment by oomichi** [comment]
Thanks for updating

/hold cancel

**Comment by oomichi** [comment]
@clarklee92 Could you rebase this again for the latest master branch?

**Comment by oomichi** [comment]
Oh, this seems unnecessary now because of https://github.com/kubernetes/kubernetes/pull/82717

/close

**Pull request closed without merging**